docker/run.sh

```diff
@@ -66,10 +66,23 @@ while [[ "$1" != "" ]]; do
     esac
 done
 
+DEV_CONTAINER_NAME=beluga_dev
+
 # Note: The `--build` flag was added to docker compose run after
 # https://github.com/docker/compose/releases/tag/v2.13.0.
 # We have this for convenience and compatibility with previous versions.
 # Otherwise, we could just forward the script arguments to the run verb.
```

Comment:
Let's remove this as it no longer applies.
```suggestion
```

Reply:
Done